draft-ietf-taps-impl.md

```diff
@@ -182,16 +179,14 @@ Aggregate [Endpoint: www.example.com:80] [Interface: Any]   [Protocol: TCP]
 |-> [Endpoint: 2001:DB8::1.80]     [Interface: LTE]   [Protocol: TCP]
 ~~~~~~~~~~
 
-Any one of these sub-entries on the aggregate connection attempt would satisfy the original application intent. The concern of this section is the algorithm defining which of these options to try, when, and in what order.
+Any one of these sub-entries on the aggregate connection attempt would satisfy the original application intent. The concern of this section is the algorithm defining which of these "Candidate" options to try, when, and in what order.
 
-During Candidate Gathering, an implementation first excludes all protocols and
-paths that match a Prohibit or do not match all Require properties.
-Then, the implementation will sort branches according to Preferred
-properties, Avoided properties, and possibly other criteria.
+During Candidate Gathering, an implementation first excludes all protocols and paths with Prohibited features as well as those which do not satisfy a Required property.
+Then, the remaining candidates will be ranked according to Preferred properties, Avoided properties, and possibly other criteria.
 
 ## Structuring Candidates as a Tree
 
-As noted above, the considereration of multiple candidates in a gathering and racing process can be conceptually structured as a tree; this terminological convention is used throughout this document.
+The considereration of multiple candidates in a gathering and racing process can be conceptually structured as a tree; this terminological convention is used throughout this document. While a tree structure is not the only way in which racing can be implemented, it does ease the illustration of how racing works. 
```

Comment:
```suggestion
The considereration of multiple candidates in a gathering and racing process can be conceptually structured as a tree; this terminological convention is used throughout this document. While a tree structure is not the only way in which racing can be implemented, it is a useful model to use to understand and illustrate connection racing.
```

Reply:
Doesn't
> a useful model to use to

sound slightly odd and redundant?

How about simply saying
> it can serve as a useful model to illustrate connection racing

Reply:
I don't understand what's slightly odd and redundant about the first version, but I guess both will work. This is for @tfpauly to answer though...